web-nfc/resources/nfc_help.js

```diff
@@ -68,14 +68,13 @@ function assertWebNDEFMessagesEqual(a, b) {
 }
 
 function testNDEFMessage(pushedMessage, watchOptions, desc) {
-  promise_test(t => {
-    return navigator.nfc.push(pushedMessage)
-      .then(() => {
-        return new Promise(resolve => {
-          navigator.nfc.watch((message) => resolve(message), watchOptions);
-        }).then((message) => {
-          assertWebNDEFMessagesEqual(message, pushedMessage);
-        });
-      });
+  promise_test(async t => {
+    let writer = new NFCWriter();
```

Comment:
Nit: s/let/const

Reply:
The same as above, we need replace all `let` with `const`, or only this one?

Reply:
All `let` could be replaced with `const`.